packages/components/constraints/src/horizontal/horizontal.js

```diff
@@ -4,60 +4,90 @@ import { css } from '@emotion/core';
 import { filterDataAttributes } from '@commercetools-uikit/utils';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 
-const getConstraintSyles = (constraint) => {
-  switch (constraint) {
-    case 'xs':
-      return css`
-        max-width: ${vars.constraintXs};
-      `;
-    case 's':
-      return css`
-        max-width: ${vars.constraintS};
-      `;
-    case 'm':
-      return css`
-        max-width: ${vars.constraintM};
-      `;
-    case 'l':
-      return css`
-        max-width: ${vars.constraintL};
-      `;
-    case 'xl':
-      return css`
-        max-width: ${vars.constraintXl};
-      `;
-    case 'scale':
-      return css`
-        width: ${vars.constraintScale};
-      `;
-    default:
-      return css``;
-  }
-};
+function getConstraintSyles({ max, constraint }) {
+  let constraintToken;
+
+  if (max === 1 || constraint === 'xs') constraintToken = vars.constraint1;
+  if (max === 2) constraintToken = vars.constraint2;
+  if (max === 3 || constraint === 's') constraintToken = vars.constraint3;
+  if (max === 4) constraintToken = vars.constraint4;
+  if (max === 5) constraintToken = vars.constraint5;
+  if (max === 6) constraintToken = vars.constraint6;
+  if (max === 7 || constraint === 'm') constraintToken = vars.constraint7;
+  if (max === 8) constraintToken = vars.constraint8;
+  if (max === 9) constraintToken = vars.constraint9;
+  if (max === 10 || constraint === 'l') constraintToken = vars.constraint10;
+  if (max === 12) constraintToken = vars.constraint12;
+  if (max === 13) constraintToken = vars.constraint13;
+  if (max === 14) constraintToken = vars.constraint14;
+  if (max === 15) constraintToken = vars.constraint15;
+  if (max === 16 || constraint === 'xl') constraintToken = vars.constraint16;
```

Comment:
I would put this into a separate function and do an early return, maybe also use a switch statement for better readability.